app/javascript/components/enrollment/steps/Identification.js

```diff
@@ -72,6 +92,63 @@ class Identification extends React.Component {
       });
   }
 
+  getLanguageOptions(languageType) {
+    const langOptions = supportedLanguages.languages.map(function(lang) {
```

Comment:
Should probably use ES6 arrow function syntax for consistency, that seems to be what is used the most on the frontend.